.gitignore

```diff
@@ -1,5 +1,7 @@
 .env
 .nyc_output
 coverage
-node_modules
+node_modules/
 npm-debug.log
+public/
```

Comment:
Could we double-check that Express doesn't throw an error if the `public` folder can't be found?

Reply:
We checked, and it works fine 💪